<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>6</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>login</string>
  </property>
  <widget class="QLabel" name="pleaseLoginLB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Please login with XYZprinting account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginPB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>240</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{ background-color: rgb(0, 150, 255); }
QPushButton:pressed{background-color: rgb(0, 150, 255); }
QPushButton {
    color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="registerPB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>240</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{ background-color: rgb(255, 0, 0); }
QPushButton:pressed{background-color: rgb(245, 0, 0); }
QPushButton {
    color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>REGISTER</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailAdressLE">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="placeholderText">
    <string> Email address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLE">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string> Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="orLB">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>300</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>or</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forgotPasswordPB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: #ff0000;
}</string>
   </property>
   <property name="text">
    <string>Forgot your password?</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="paletteLB">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BCPware for
PartPro350 xBC</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="rememberMeCB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Remember me</string>
   </property>
  </widget>
  <widget class="QLabel" name="bcp_icon">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/icons/bcp_icon.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="eyeCB">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
